src/RulesTest/RuleImplementations/RuleWithPassesTestNotImplemented.cs

```diff
@@ -18,7 +18,8 @@ public RuleWithPassesTestNotImplemented() : base()
 
         public override EvaluationCode Evaluate(IA11yElement element)
         {
-            throw new NotImplementedException();
+            // Don't throw an exception here so we can be sure the only exception thrown is in Rule.PassesTest
+            return EvaluationCode.Pass;
```

Comment:
General comment about this class: Would it be worth a comment that the entire class will go away once the migration to PassesTest is complete?